setup.py

```diff
@@ -271,6 +271,7 @@ def get_exts_for(name):
         "xmltodict>=0.12",
         "ipaddress; python_version<'3.7'",
         "envier",
+        "defusedexpat>=0.4",
```

Comment:
are we sure this is more secure? it seems unmaintained (last activity was from 2013)